anda/tools/F-Sy-H/anda.hcl

```diff
@@ -0,0 +1,5 @@
+project pkg {
```

Comment:
```suggestion
project pkg {
  arches = ["x86_64"]
```

Reply:
?

Reply:
ah in case of the confusion, since this is a noarch package, we can just build this with the x86_64 builders. if we use both builders there will be duplications

Reply:
Sorry I thought I commited it.